msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -814,7 +814,7 @@ When a group install scope is selected, it defines the default capability when t
 |Name| Type| Maximum size | Required | Description|
 |---|---|---|---|---|
 |`team`|String|||When the install scope selected is `team`, this field specifies the default capability available. Options: `tab`, `bot`, or `connector`.|
-|`groupChat`|String|||When the install scope selected is `groupChat`, this field specifies the default capability available. Options: `tab`, `bot`, or `connector`.|
```

Comment:
Group chat is reference in multiple place. Please fix it.

![image](https://github.com/user-attachments/assets/da5eb5a8-42f7-4240-853f-1bcccb216ff8)

Reply:
Hi @Wajeed-msft 
In the current schema v1.17, groupchat is in lower case only for the property [`defaultGroupCapability`](https://teams.microsoft.com/l/message/19:9da4360914d24cf58a6a0c54af6c6211@thread.skype/1723532567238?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=32e3b156-66b2-4135-9aeb-73295a35a55b&parentMessageId=1718105311068&teamName=Teams%20Platform&channelName=Platform%20Doc-Queries&createdTime=1723532567238) while all the other instances are in camelcase. We have [requested the team](https://teams.microsoft.com/l/message/19:9da4360914d24cf58a6a0c54af6c6211@thread.skype/1726811029876?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=32e3b156-66b2-4135-9aeb-73295a35a55b&parentMessageId=1718105311068&teamName=Teams%20Platform&channelName=Platform%20Doc-Queries&createdTime=1726811029876) to fix it in the next schema releases. But we need to update the doc to support the current schema version.